ui/packages/consul-ui/tests/unit/mixins/creating-route-test.js

```diff
@@ -1,12 +0,0 @@
-import EmberObject from '@ember/object';
-import CreatingRouteMixin from 'consul-ui/mixins/creating-route';
-import { module, test } from 'qunit';
-
-module('Unit | Mixin | creating route', function() {
-  // Replace this with your real tests.
-  test('it works', function(assert) {
-    let CreatingRouteObject = EmberObject.extend(CreatingRouteMixin);
-    let subject = CreatingRouteObject.create();
-    assert.ok(subject);
-  });
-});
```

Comment:
With this change I could finally get rid of a load of our mixins, again this will help us get onto the next ember LTS.